ncm-metaconfig/src/main/metaconfig/keepalived/pan/schema.pan

```diff
@@ -39,27 +39,38 @@ type keepalived_service_vrrpinstance_config = {
     'interface' : string
 };
 
-@documentation {
+@documentation{
     The vrrp_instance section
 }
 type keepalived_service_vrrpinstance = {
     'name' : string
     'config' : keepalived_service_vrrpinstance_config
     'virtual_ipaddresses' : keepalived_service_vip[]
-    'track_scripts' : string[]
+    'track_scripts' ? string[]
     'unicast_peer' ? type_ip[]
     'unicast_src_ip' ? type_ip
     'virtual_routes' ? string[]
     'track_interface' ? string[]
 };
 
-@documentation {
+@documentation{
+    The vrrp_sync_group section
+}
+type keepalived_service_vrrpsyncgroup = {
+    'group' : string[]
+    'notify_master' ? string
+    'notify_backup' ? string
+    'notify_fault' ? string
```

Comment:
These are scripts with possible arguments right?
Could/should they be modelled with a type?
```
type keepalived_notify_script = {
    script : absolute_file_path
    args ? string_trimmed[]
};
```

```suggestion
    'notify_master' ? keepalived_notify_script
    'notify_backup' ? keepalived_notify_script
    'notify_fault' ? keepalived_notify_script
```